modules/concurrency/include/hephaestus/concurrency/spinner.h

```diff
@@ -40,27 +43,29 @@ class Spinner {
   auto stop() -> std::future<void>;
   void wait();
 
-  /// @brief  Set a callback that will be called when the spinner is stopped.
-  /// This callback could be extendend to pass the reason why the spinner was stopped, e.g. exceptions, ...
-  void setTerminationCallback(Callback&& termination_callback);
```

Comment:
This function is used by the SpinnerManager. If we remove it we cannot use the Spinner manager. this is because the spinner manager receive a spinner and need to be able to inject a termination callback,